webpack.config.js

```diff
@@ -59,32 +59,32 @@ export default {
     new CopyWebpackPlugin({
       patterns: [
         {
-          from: 'schemas/fingerprint-server-api.yaml',
+          from: 'schemas/fingerprint-server-api-for-sdks.yaml',
           to: 'schemas/fingerprint-server-api.yaml',
           transform: (content) => transformSchema(content),
         },
         {
-          from: 'schemas/fingerprint-related-visitors-api.yaml',
-          to: 'schemas/fingerprint-related-visitors-api.yaml',
+          from: 'schemas/fingerprint-related-visitors-api-readme-explorer.yaml',
+          to: 'schemas/fingerprint-related-visitors-api-readme-explorer.yaml',
```

Comment:
@JuroUhlar could we simply change the artefact name?

Reply:
What do you mean? remove transformers?